CHANGELOG.md

```diff
@@ -24,59 +48,51 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
-- Add feature flag bEnableInitialOnlyReplicationCondition for COND_InitialOnly support.
+- Add feature flag `bEnableInitialOnlyReplicationCondition` for `COND_InitialOnly` support.
 - Added a function that allows the worker coordinator to periodically restart the simulated player clients with a bunch of parameters. This feature is disabled by default and can be enabled via `max_lifetime` setting.
-  - When you define your test scenario yaml file or call `StartCoordinator.sh`, add args as below
-  - `max_lifetime=90` will let your sim player clients lives no longer than 90 minutes.
-  - `min_lifetime=30` will let your sim player clients lives at least 30 minutes.
-  - `use_new_simulated_player=1` will use a new sim player id everytime when you start a sim player client.
+  - When you define your test scenario yaml file or call `StartCoordinator.sh`, please provide the arguments as below:
+  - `max_lifetime=90` will cap your simulated player clients' lives at 90 minutes.
+  - `min_lifetime=30` will allow your simulated player clients to live for at least 30 minutes.
+  - `use_new_simulated_player=1` will use a new simulated player id everytime that you start a simulated player client.
 - Exposing worker upstream/downstream window sizes as GDK options for both clients and servers, (`ClientDownstreamWindowSizeBytes`, `ClientUpstreamWindowSizeBytes`) and (`ServerDownstreamWindowSizeBytes` and `ServerUpstreamWindowSizeBytes`).
-- `bOnlyRelevantToOwner` is now supported. Ownership must be setup prior to the first replication of the Actor otherwise it will be ignored.
-- GDK heartbeat settings are now used to control the worker heartbeat configurations.
+- `bOnlyRelevantToOwner` is now supported. Ownership must be setup prior to the first replication of the `Actor` otherwise it will be ignored.
 - Added a property to specify the test settings overrides config filename in the `World Settings` so that maps can share config files during automated testing. This replaces the option to automatically use the map name to determine the config filename.
-- Added a setting to control which CrossServer RPC implementation is used. Both feature mentioned below are only enabled when the RoutingWorker is the chosen implementation. Spatial commands are still the default for now.
-- Added reliable CrossServer RPC. Reliable CrossServer RPC now require a sender actor which will be the reference point for ordering in a multi-worker environment. An additional UFUNCTION Tag, Unordered, was added to opt-out of this requirement.
-- Added NetWriteFence UFUNCTION Tag. This tag is used when Network writes to an actor should be ordered with regard to updates to another actor. This is relevant in worker recovery/snapshot reloading to get some ordering guarantees when SpatialOS can write updates to entities in any order.
+- Added a `-FailOnNetworkFailure` flag that makes a Spatial-enabled game fail on any NetworkFailure.
```

Comment:
```suggestion
- Added `URemotePossessionComponent` to deal with Cross-Server Possession. Add this componenet to an AController, it will possess the Target Pawn after OnAuthorityGained. It can be implemented in C++ and Blueprint.
```